iXBRLViewerPlugin/__init__.py

```diff
@@ -62,6 +60,15 @@ def iXBRLViewerCommandLineOptionExtender(parser, *args, **kwargs):
     # Force logging to use a buffer so that messages are retained and can be
     # retrieved for inclusion with the viewer.
     parser.add_option("--logToBuffer", action="store_true", dest="logToBuffer", default=True, help=argparse.SUPPRESS)
+    parser.add_option("--use-stub-viewer",
+                      action="store_true",
+                      dest="useStubViewer",
+                      help="Use stub viewer for faster loading of inspector (requires web server)")
+    parser.add_option("--viewer-suffix",
```

Comment:
Should we add these options to the build_viewer.py file?

Reply:
Done in [c8a59dd](https://github.com/Workiva/ixbrl-viewer/pull/394/commits/c8a59dd60ca61c8619898a2c7a422c377c83f6e7)